gm4_calling_bell/data/gm4_calling_bell/functions/trader/summon.mcfunction

```diff
@@ -0,0 +1,12 @@
+# summon a wandering trader for the player
+# @s = marker, tag=gm4_wandering_trader_ray
```

Comment:
This is incorrect; it's run as the player, at the marker/bell. Player context is saved here, so there's no need for `@a`